.github/workflows/pythontest.yml

```diff
@@ -0,0 +1,31 @@
+name: Python testing
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
I'm not familiar with the difference, but can it be `v1`? Looks like the official documents are suggesting `v2`, so I'd like to know if you have any special reasons for this. https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions

Reply:
good catch! 00cd327dc584bda64d8ad3108191984f0e8ec21a